package.json

```diff
@@ -1,12 +1,12 @@
 {
     "private": true,
     "scripts": {
-        "dev": "yarn development",
+        "dev": "yarn run development",
```

Comment:
you can actually run scripts with `yarn` without the `run` keywork, it does not work with npm only